app/screens/image_preview/downloader.ios.js

```diff
@@ -183,18 +156,58 @@ export default class Downloader extends PureComponent {
         }
 
         return (
-            <View style={styles.progressContent}>
-                {component}
-                <View style={styles.bottomContent}>
-                    {savedComponent}
+            <View style={styles.bottomContent}>
+                {savedComponent}
+            </View>
+        );
+    };
+
+    renderProgress = (fill) => {
+        const {isVideo} = this.state;
+        const realFill = Number(fill.toFixed(0));
+
+        let component;
+        if (realFill === 100) {
+            component = (
+                <Icon
+                    name='ios-checkmark'
+                    size={64}
+                    color='white'
+                />
+            );
+        } else {
+            component = (
+                <View style={styles.progressCirclePercentage}>
+                    <Text style={styles.progressText}>
+                        {`${fill.toFixed(0)}%`}
+                    </Text>
+                    {!isVideo &&
+                    <TouchableOpacity
+                        style={styles.cancelButton}
+                        onPress={this.downloadDidCancel}
+                    >
+                        <FormattedText
+                            id='channel_modal.cancel'
+                            defaultMessage='Cancel'
+                            style={styles.cancelText}
+                        />
+                    </TouchableOpacity>
+                    }
                 </View>
+            );
+        }
+
+        return (
+            <View>
+                {component}
             </View>
+
         );
     };
 
     renderStartDownload = () => {
         return (
-            <View style={styles.progressContent}>
+            <View>
```

Comment:
Does this View do anything any more, or can it be removed?

Reply:
Without this View it does not show